### DISCUSSION TITLE
How to downgrade vps

### DISCUSSION THREAD

**Discussion by lucairr** [Ysurac/openmptcprouter#2608] [Q&A]
Hi, there I updated my vps to 0.1028 few weeks ago, but with this version and omr updated to 0.59, I'm having some trouble with the aggregation of my two wan.
There is a way to downgrade the vps to 0.1026?
Thanks

**Comment by Ysurac** [comment]
downgrading is not supported.

**Reply by lucairr** [reply]
And It's not even possible to install 0.1026 on a new vps?

**Reply by Ysurac** [reply]
You can do what you want if you find how, but it's not supported and no help on this.
There is no many change on VPS side for kernel 5.4 between 0.1026 and 0.1028, only a kernel update that fix MPTCP issues.
